### PR TITLE
Forus frontend fixes

### PR DESCRIPTION
**Pull request by GerbenBosschieter**
- Some financial dashboard fixes
- No voucher  -> open activate code popup

### CODE REVIEW

**Review by GerbenBosschieter — COMMENTED** [review]
Commit 
662bb67 overwrites 
6553d2c

**Comment by maxvisser** [comment]
Is 'ontdek hoe' ook vervangen door activeer je activatiecode?

**Comment by GerbenBosschieter** [comment]
nee, I will do this.

File: src/sources/forus-webshop-zuidhorn/js/angular-1/i18n/i18n-en.js [hunk 1]
```diff
@@ -362,6 +363,7 @@ module.exports = {
             mail: "Heb je een brief van de gemeente ontvangen met een activatiecode en deze nog niet gebruikt? Ga dan verder. Heb je deze al wel gebruikt? Sluit dan dit venster en klik op 'login'.",
             link: "Vul je e-mailadres in om een link te ontvangen waarmee je kunt inloggen.",
             code: "Vul de activatiecode in die je per brief hebt ontvangen.",
+            voucher_email: "Het is gelukt!",
```
Comment (danrminds) [inline src/sources/forus-webshop-zuidhorn/js/angular-1/i18n/i18n-en.js:366]: Max hates this

**Review by danrminds — APPROVED** [review]

**Pull request merged**